ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/test/java/de/adorsys/ledgers/baam/db/domain/SeniorManagerAccessTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2018-2023 adorsys GmbH and Co. KG
+ * All rights are reserved.
+ */
+
+package de.adorsys.ledgers.baam.db.domain;
+
+import org.junit.jupiter.api.Test;
+class SeniorManagerAccessTest {
+
+    @Test
+    void createManagerRole() {
+    }
+
+    @Test
+    void modifyManagerRole() {
+    }
+
+    @Test
+    void revokeManagerRole() {
+    }
+
+    @Test
+    void suspendAccess() {
+    }
+
+    @Test
+    void activateAccess() {
+    }
```

Comment:
PLease right a full test for these entity..

use you can use an **h2 database**  which acts as an in memory DB for the test..

these is to ensure that the entire logic is working perfectly fine and everything is actually stored in a database.

Also please ensure to do a **git pull** so as to see recent changes, these will help you see the new recent project structure for our module.